Examples/WildeGuess/WildeGuess/Components/InteractiveQuoteComponent.mm

```diff
@@ -23,24 +23,33 @@
 @implementation InteractiveQuoteComponent
 {
   CKComponent *_overlay;
+  Quote *_quote;
+  QuoteContext *_context;
 }
 
 + (instancetype)newWithQuote:(Quote *)quote
                      context:(QuoteContext *)context
 {
-  CKComponentScope scope(self);
-  const BOOL revealAnswer = [scope.state() boolValue];
-
   CKComponent *overlay =
-  revealAnswer
-  ? [SuccessIndicatorComponent
-     newWithIndicatesSuccess:[quote.author isEqualToString:oscarWilde]
-     successText:[NSString stringWithFormat:@"This quote is by %@", oscarWilde]
-     failureText:[NSString stringWithFormat:@"This quote isn't by %@", oscarWilde]]
-  : nil;
+  [SuccessIndicatorComponent
```

Comment:
why is this component created in +new instead of render?

Reply:
Just because it was saved as a prop before. But it actually makes the animation incorrect in this case as this component was created only on a specific state.

We should move it to the render method and alternate the animation method to receive the state in order to solve this issue.